package/src/frontend/media/media.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'underscore';
+import {createMediaPlayer} from './createMediaPlayer';
+
+export const media = {
+  players: {},
+  mute: function (value) {
+    _.each(Object.values(this.players), (player) => {
+      player.muted(value);
+    });
+  },
+  processSource: function (fileSource, type) {
+    return [
+      {type: type+'/ogg', src: `${fileSource.urls.ogg}?u=1`},
+      {type: type+'/mp4', src: `${fileSource.urls.m4a}?u=1`},
+      {type: type+'/mp3', src: `${fileSource.urls.mp3}?u=1`}
+    ];
+  },
```

Comment:
Basically this was done in my old media ticket that sources were prepared in react component and then passed to MediaTag. Same is done in existing pageflow-react setup.
Since we don't have MediaTag now so we can simply pass the sources to player.src(sources) method.
after this processSources method is not required anymore